packages/@react-spectrum/actionbar/package.json

```diff
@@ -31,7 +31,6 @@
     "url": "https://github.com/adobe/react-spectrum"
   },
   "dependencies": {
-    "@adobe/react-spectrum": "^3.14.0",
```

Comment:
good find, that probably should not be there. looks like it snuck in at the beginning and we never noticed
I don't see anything that would need it